src/nxt_http_return.c

```diff
@@ -92,7 +92,7 @@ nxt_http_return(nxt_task_t *task, nxt_http_request_t *r,
         nxt_tstr_str(conf->location, &loc);
     }
 
-    nxt_debug(task, "http return: %d (loc: \"%V\")", conf->status, &loc);
+    nxt_debug(task, "http return: %d (location: \"%V\")", conf->status, &loc);
```

Comment:
"http response"

Reply:
We use `"return"` in the config, so it makes sense to be consistent with that.  Also, "response" refers to the entire HTTP response, including body and headers, right?  "status code" is what we mean here, right?  Do we want return, status code, or response?